sc/social/like/tests/test_subscribers.py

```diff
@@ -21,10 +21,13 @@
 from sc.social.like.utils import MSG_INVALID_OG_DESCRIPTION
 from sc.social.like.utils import MSG_INVALID_OG_LEAD_IMAGE_DIMENSIONS
 from sc.social.like.utils import MSG_INVALID_OG_TITLE
+from testfixtures import log_capture
 from zope import schema
 from zope.component import getUtility
 from zope.event import notify
 
+import logging
+import requests_mock
```

Comment:
this is new too!

Reply:
we're using it in collective.behavior.amp.